### DISCUSSION TITLE
I have an Issue using and understanding how 'core-js' works with parcel

### DISCUSSION THREAD

**Discussion by VelynnXV** [parcel-bundler/parcel#5881] [Q&A]
Hello,

I have a very simple app that i want to make it work with ie11, but im stuck for days trying different solutions from stackOverflow, babel documentation, parcel documentation and so on without any results. I am not sure what I am missing in my configuration.

package.json
```json
{
  "name": "mypackage",
  "version": "1.0.0",
  "description": "This is my package, using Parcel. Parcel is a web application bundler, differentiated by its developer experience. It offers blazing fast performance utilizing multicore processing, and requires zero configuration.",
  "app": "public/index.html",
  "scripts": {
    "dev": "npm run clean && parcel public/index.html --dist-dir development -p 3000",
    "build": "rimraf ./dist && parcel build public/*.html --dist-dir dist --public-url ./ --no-optimize",
    "clean": "rimraf ./development && rimraf ./.parcel-cache"
  },
  "browserslist": [
    "IE 11",
    "> 1%"
  ],
  "author": "velynn",
  "license": "UNLICENSED",
  "devDependencies": {
    "@parcel/babel-plugin-transform-runtime": "^2.0.0-nightly.1823",
    "@parcel/babel-preset-env": "^2.0.0-alpha.3",
    "@parcel/optimizer-cssnano": "^2.0.0-nightly.598",
    "@parcel/optimizer-htmlnano": "^2.0.0-nightly.598",
    "@parcel/packager-css": "^2.0.0-nightly.598",
    "@parcel/packager-html": "^2.0.0-nightly.598",
    "@parcel/transformer-css": "^2.0.0-nightly.598",
    "@parcel/transformer-html": "^2.0.0-nightly.598",
    "@parcel/transformer-inline-string": "^2.0.0-nightly.598",
    "@parcel/transformer-postcss": "^2.0.0-nightly.598",
    "@parcel/transformer-posthtml": "^2.0.0-nightly.598",
    "@parcel/transformer-sass": "^2.0.0-nightly.598",
    "@webcomponents/webcomponentsjs": "^2.5.0",
    "core-js": "^3.9.0",
    "parcel": "^2.0.0-beta.1",
    "postcss": "^8.2.6",
    "sass": "^1.32.7"
  },
  "dependencies": {}
}

```

.babelrc
```
{
  "presets": [
    ["@parcel/babel-preset-env", {
      "targets": {
        "ie": "11"
      },
      "useBuiltIns": "entry",
      "corejs": 3.8
    }]
  ],
  "plugins": ["@parcel/babel-plugin-transform-runtime"]
}
```

Ive read somewhere that i needed to add this in my app.js 

> import 'core-js/stable'
> import "regenerator-runtime/runtime";

in order to make pollyfills work, but when i build and run my app i get : 

> Uncaught ReferenceError: $a9cff48d0a9a1d5e6d3a39756f182d0e$exports is not defined
>     at es.map.js:3

Is there any simple solution to make an app work in ie11? I have never seen a working example of a application made with parcel that works with babel. Maybe that can help me understand how to setup my app.

My source code :https://github.com/VelynnXV/Front-End-Workflow

the website: https://nifty-noether-cafbd5.netlify.app/

**Comment by FDiskas** [comment]
Did you followed documentation at https://github.com/zloirock/core-js
?
Also last error complains something about `export` does your source code was successfully compiled and does not contain nodejs syntax's?
